tests/para-tests/moonbase/test-runtime-upgrade.ts

```diff
@@ -98,6 +98,8 @@ describeParachain(
         );
 
         await context.upgradeRuntime({ runtimeName: "moonbase", runtimeTag: RUNTIME_VERSION });
+        process.stdout.write("Waiting extra block for storage values to update...\n");
+        await context.waitBlocks(1);
```

Comment:
I think the fix for this should be in the upgradeRuntime.
I'm surprises it doesn't pass as I was manually testing it without trouble. let me quicklyu troubleshoot it